storey/windowed_store.py

```diff
@@ -74,7 +78,9 @@ async def _do(self, event):
 
         element = event.body
         key = event.key
-        event_time = event.time
+        event_time = event.processing_time
+        if self._time_field:
+            event_time = event.body.pop(self._time_field)
```

Comment:
How does that work if `_time_field` is an `EmitPolicy`?

Reply:
Why would `_time_field` be an `EmitPolicy`? They're unrelated things.

Reply:
See line 39 above: 
```
time_field: Union[str, EmitPolicy, None] = None,
```
Maybe the issue is there then?

Reply:
Yup, not sure how that happened...